src/tests/libecalc/presentation/yaml/test_venting_emitter.py

```diff
@@ -134,9 +130,18 @@ def test_venting_emitter_oil_volume(variables_map):
     }
     emissions_ch4 = emission_result["ch4"]
 
-    regularity_evaluated = float(
-        Expression.evaluate(regularity[datetime(1900, 1, 1)], fill_length=1, variables=variables_map.variables)
-    )
+    try:
+        regularity_array = Expression.evaluate(
+            regularity[datetime(1900, 1, 1)], fill_length=1, variables=variables_map.variables
+        )
+
+        regularity_evaluated = float(regularity_array[0])
+
+    except IndexError as e:
+        raise IndexError("Failed to evaluate regularity: array index out of range.") from e
+
+    assert regularity_evaluated == regularity_expected
+
     expected_result = [oil_value * regularity_evaluated * emission_factor / 1000 for oil_value in oil_values()]
```

Comment:
```suggestion
    expected_result = [oil_value * expected_regularity * emission_factor / 1000 for oil_value in oil_values()]
```

Reply:
Yes, to not evaluate the regularity at all. Probably the best yes. And the evaluation etc. is tested separately. Ok, will change it.